mentat/terminal/prompt_session.py

```diff
@@ -57,6 +59,23 @@ def prompt_continuation(
             "" if is_soft_wrap else [("class:continuation", " " * (width - 2) + "> ")]
         )
 
+    def _toggle_audio(self):
```

Comment:
Is there a specific reason to make Whisper a terminal only feature? While it does make more sense for a client to be handling the input, we could always have the server run whisper and have a channel that the server can use to send input (rather than output) to the client. That way we could run whisper on any client, although it obviously is a bit weird having the server type stuff in for the user haha.

Reply:
When I started this I thought it made more sense to do it at the input layer and therefore on the client side. I couldn't think of a natural way to expose it in the session for any client to use.

But all my future enhancement ideas would be a lot easier to do on the server side so maybe you're right that I should switch it.